x-pack/plugins/session_view/public/components/session_view/index.test.tsx

```diff
@@ -135,5 +136,49 @@ describe('SessionView component', () => {
         expect(renderResult.getAllByTestId('sessionView:sessionViewRefreshButton')).toBeTruthy();
       });
     });
+
+    describe('TTYPlayer button', () => {
+      it('should show tty player button, if session has output', async () => {
+        mockedApi.mockImplementation(async (options) => {
+          // for some reason the typescript interface for options says its an object with a field called path.
+          // in reality options is a string (which equals the path...)
+          const path = String(options);
```

Comment:
Would this do the trick?

```js
mockedApi.mockImplementation(async (path : any) => {
```

Usually, we don't need to worry about types in tests that much (when dealing with jest mocks), so an assertion like this is fine (or even a // @ts-ignore on the line before the error)